framework/logging.cpp

```diff
@@ -435,12 +439,55 @@ static uint8_t status_level(char letter)
         return LOG_LEVEL_VERBOSE(2);    /* info only with -vv */
     case 'd':
         return LOG_LEVEL_VERBOSE(3);    /* debug only with -vvv or the log file */
+    case 'S':
+        return LOG_LEVEL_QUIET;
     }
 
     log_warning("got improper status log message '%c'", letter);
     return 2;
 }
 
+static char skip_category_to_char(SkipCategory category){
+
+    switch (category) {
+    case SkipCategory::RESOURCE_UNAVAILABLE:
+        return 65;
+    case SkipCategory::CPU_NOT_SUPPORTED:
+        return 66;
+    case SkipCategory::DEVICE_NOT_FOUND:
+        return 67;
+    case SkipCategory::DEVICE_NOT_CONFIGURED:
+        return 68;
+    case SkipCategory::OTHERS:
+        return 69;
+    case SkipCategory::RUNTIME_SKIP:
+        return 70;
+    }
+
+    log_warning("No category present, please select one from the predefined ones");
```

Comment:
"Please select" does not apply to this situation, it's not interactive.

Reply:
Working on this. The new version doesn't have this function at all.